anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -447,7 +447,8 @@ def evaluate(self, image_obj, context):
                                     parameter_data[
                                         "link"
                                     ] = "{}/query/vulnerabilities?id={}".format(
-                                        api_endpoint, vulnerability_cpe.vulnerability_id
+                                        api_endpoint,
+                                        vulnerability_cpe.parent.normalized_id,
```

Comment:
Rollback please. Refer to https://github.com/anchore/anchore-engine/pull/843/files#r558711037